tutorials/misc/plot_report.py

```diff
@@ -195,3 +195,4 @@
 ###############################################################################
 # With the context manager, the updated report is also automatically saved
 # back to :file:`report.h5` upon leaving the block.
+# This line needs to be removed.
```

Comment:
FYI instead of doing this, which will require another commit (and another round of CIs), it's better to either improve the wording/tutorial in some way, or (if you can't easily find an improvement) do some harmless (PEP8-compatible) newline removal or insertion.

Reply:
Also in a commit where all you care about is Circle running, it helps to do `[skip travis]` in the commit message since Travis is usually our CI bottleneck (and Azure doesn't have an equivalent)

Reply:
Thanks @larsoner, I will note this down :)